docs/internal-documentation/maintenance/Garbage-Collection-Testing.md

```diff
@@ -0,0 +1,122 @@
+# Testing instructions for the Garbage Collection feature of the Zonemaster Backend module
+
+## Introduction
+The purpose of this instruction is to serve as a notice for manual testing of the new garbage collection feature at release time.
+
+## Testing the unfinished tests garbage collection feature
+
+1. Ensure that the database has the required aditionnal columns for this feature:
+```
+SELECT nb_retries FROM test_results LIMIT 0;
+```
+Should return:
+
+```
+ nb_retries 
+------------
+(0 rows)
+```
+
+2. Check that your backend_config.ini has the proper parameter set
```

Comment:
2. Check that your `/etc/zonemaster/bacend_config.ini` (or, in FreeBSD, `/usr/local/etc/zonemaster/bacend_config.ini`) has the proper parameter set

Reply:
Fixed in the commit 65f5257